var/spack/repos/builtin/packages/margo/package.py

```diff
@@ -7,43 +7,49 @@
 
 
 class Margo(AutotoolsPackage):
-    """Argobots bindings to the Mercury RPC API"""
+    """A library that provides Argobots bindings to the Mercury RPC
+    implementation.  This name will be deprecated soon; please use the
+    mochi-margo package instead."""
 
-    homepage = "https://xgitlab.cels.anl.gov/sds/margo"
-    url      = "https://xgitlab.cels.anl.gov/sds/margo/-/archive/v0.4.3/margo-v0.4.3.tar.gz"
-    git      = "https://xgitlab.cels.anl.gov/sds/margo.git"
+    homepage = 'https://xgitlab.cels.anl.gov/sds/margo'
+    git = 'https://xgitlab.cels.anl.gov/sds/margo.git'
 
-    maintainers = ['fbudin69500', 'chuckatkins', 'carns']
+    maintainers = ['carns', 'mdorier']
 
+    version('master', branch='master')
     version('develop', branch='master')
-    version('0.4.3', sha256='61a634d6983bee2ffa06e1e2da4c541cb8f56ddd9dd9f8e04e8044fb38657475')
-
-    variant('shared', default=True,
-            description='Build shared libraries instead of static libraries')
-
-    depends_on('autoconf',  type='build')
-    depends_on('automake',  type='build')
-    depends_on('libtool',   type='build')
-    depends_on('m4',        type='build')
-    depends_on('pkgconfig', type='build')
-
-    depends_on('mercury')
+    version('0.9', tag='v0.9')
+    version('0.7.2', tag='v0.7.2')
+    version('0.7.1', tag='v0.7.1')
+    version('0.7', tag='v0.7')
+    version('0.6.4', tag='v0.6.4')
+    version('0.6.3', tag='v0.6.3')
+    version('0.6.2', tag='v0.6.2')
+    version('0.6.1', tag='v0.6.1')
+    version('0.6', tag='v0.6')
+    version('0.5.2', tag='v0.5.2')
+    version('0.5.1', tag='v0.5.1')
+    version('0.5',   tag='v0.5')
+    version('0.4.7', tag='v0.4.7')
+    version('0.4.6', tag='v0.4.6')
+    version('0.4.5', tag='v0.4.5')
+    version('0.4.4', tag='v0.4.4')
+    version('0.4.3', tag='v0.4.3')
+    version('0.4.2', tag='v0.4.2')
+
+    depends_on('json-c', when='@0.9:')
+    depends_on('autoconf@2.65:', type=("build"))
+    depends_on('m4', type=('build'))
+    depends_on('automake', type=("build"))
+    depends_on('libtool', type=("build"))
+    depends_on('pkg-config', type=("build"))
```

Comment:
```suggestion
    depends_on('pkgconfig', type=("build"))
```
Spack has a virtual package for this

Reply:
Thanks @adamjstewart , I'll make these changes later today.  I'll probably drop the @develop version here and just leave the @master one, though; there is no develop branch in the source repo.

Reply:
I just pushed the requested revisions @adamjstewart I just pushed the revisions if you want to have a look.

Reply:
whoops modulo formatting goof, now I got it :)

Reply:
Actually there is a CI problem here that I don't fully understand https://github.com/spack/spack/pull/20273/checks?check_run_id=1749955042

Reply:
This should be fixed on the `develop` branch, let me close and reopen to trigger ci again